scripts/test/MariReduction.py

```diff
@@ -23,6 +23,14 @@
     mpl = None
 
 
+def reduced_filename(runs, ei, is_sum, prefix=None):
```

Comment:
How did you know how to implement this?

Reply:
I discovered that the missing function was in the mantid scriptsrepository
https://github.com/mantidproject/scriptrepository/blob/3c8ebfbd0dc49127b496c69c0bc08a2ebb555ca6/direct_inelastic/MARI/MARIReduction_Sample.py#L467

Reply:
nice